serverless/src/util/aws/doesObjectExist.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * check if an object exists
+ * see https://stackoverflow.com/questions/26726862/how-to-determine-if-object-exists-aws-s3-node-js-sdk
+ * @param {Object} s3 - instance of AWS.S3 from 'aws-sdk'
+ * @param {String} bucketName - name of the S3 bucket
+ * @param {String} key - object key
+ */
+export default async function doesObjectExist(s3, bucketName, key) {
+  if (!bucketName) throw Error('missing bucketName')
```

Comment:
I think we use sentence case for errors elsewhere in the app. We should be consistent if thats the case.

Reply:
fixed with https://github.com/nasa/earthdata-search/pull/1630/commits/572256e69b5c0cd0d2be6ab3d7e5c44fe288cc2f